code/modules/client/preferences.dm

```diff
@@ -761,6 +771,68 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					return job_engsec_low
 	return 0
 
+/datum/preferences/proc/SetTraits(mob/user)
+	if(!SStraits)
+		to_chat(user, "<span class='danger'>The trait subsystem is still initializing! Try again in a minute.</span>")
+		return
+	if(SSticker.HasRoundStarted() && !isnewplayer(user))

```

Comment:
Why. This is inconsistent with any other preferences.

Reply:
If someone is cloned during a round I suspect that it'd change their traits midround based on what they changed between dying and being cloned, due to how body transference works.

Reply:
Just cache whatever they have at the start of the round.